repo.json

```diff
@@ -4970,6 +4970,24 @@
           }
         }
       ]
+    },
+    {
+      "id": "woutervh-mapbox-panel",
+      "type": "panel",
+      "url": "https://github.com/faxm0dem/grafana-riemann-websocket-datasource",
```

Comment:
Should this be https://github.com/woutervh-/grafana-mapbox?

Reply:
Oops, you're right, I'll fix it.